impl/src/main/java/org/ehcache/loaderwriter/writebehind/AbstractWriteBehindQueue.java

```diff
@@ -519,28 +467,10 @@ private void processSingleOperation(List<SingleOperation<K, V>> quarantinedItems
       SingleOperation<K, V> item = quarantinedItems.get(0);
       LOGGER.debug("{} : processItems() : processing {} ", getThreadName(), item);
 
-      int executionsLeft = retryAttempts + 1;
-      while (executionsLeft-- > 0) {
-        try {
-          item.performSingleOperation(cacheLoaderWriter);
-          break;
-        } catch (Exception e) {
-          if (executionsLeft <= 0) {
-            LOGGER.warn("Exception while processing key '{}' write behind queue : {}", item.getKey(), e);
-            //TODO : How to handle this properly 
-            if(cacheLoaderWriter instanceof ResilientCacheWriter ) {
-              ((ResilientCacheWriter<K, V>) cacheLoaderWriter).throwAway(item.getKey(), item instanceof WriteOperation ? ((WriteOperation<K, V>)item).getValue() : null,  e);
-            }
-          } else {
-            LOGGER.warn("Exception while processing write behind queue, retrying in {} seconds, {} retries left : {}", retryAttemptDelaySeconds, executionsLeft, e);
-            try {
-              Thread.sleep(retryAttemptDelaySeconds * MS_IN_SEC);
-            } catch (InterruptedException e1) {
-              Thread.currentThread().interrupt();
-              throw new Exception("Exception while processing key '" + item.getKey() + "' write behind queue", e);
-            }
-          }
-        }
+      try {
+        item.performSingleOperation(cacheLoaderWriter);
+      } catch (Exception e) {
+        LOGGER.warn("Exception while processing key '{}' write behind queue : {}", item.getKey(), e);
```

Comment:
I believe we need to delegate to the resilience strategy here and by default remove the mapping from the cache.